learnEcl/3100-table.md

```diff
@@ -2,3 +2,232 @@
 title: Table
 slug: table
 ---
+
+# TABLE
+
+TABLE is the most commonly-used data aggregation functions in ECL. It creates a new dataset in memory while workunit is running. The new table inherits the implicit rationality the recordset has (if any), unless the optional expression is used to perform aggregation. There are two forms  of Table usage:
```

Comment:
"while workunit" -> "while a workunit"

I would suggest deleting the entire sentence of "The new table inherits..." because it is confusing for non-experts.

"two forms of Table usage" -> "two forms of TABLE usage"